docs/semantics.md

```diff
@@ -0,0 +1,87 @@
+# RDF-star Semantics Options
+
+There have been several different semantics proposals for quoted triples, some with multiple semantics.  The proposed semantics differ in technical details but also differ in the entailments they support.  The major differences between semantics can be summarized in whether they are sensitive to the precise syntax of quoted triples or only to the semantics of the components of quoted triples.   The first kind of semantics have been called opaque and the second kind transparent.  There is also a semi-opaque (or semi-transparent) semantics, as in the community report, where the syntax of IRIs and literals are important but only the meaning of blank nodes is.
+
+The difference between the semantics can be seen in the following examples.
+
+In RDF (extended to include owl:sameAs as equality to provide a way of requiring that two different IRIs mean the same thing)
+```
+:b owl:sameAs :c .
+_:a :b "4"^^xsd:int .
+```
+entails
+```
+_:f :b "4"^^xsd:int .
+```
+and 
+```
+_:a :b "4"^^xsd:integer .
+```
+and 
+```
+_:a :c "4"^^xsd:integer .
+```
+
+
+## Transparent Semantics
+
+In the transparent semantics
+```
+:b owl:sameAs :c .
+<<_:a :b "4"^^xsd:int >> :d :e .
+```
+entails
+```
+<<_:f :b "4"^^xsd:int >> :d :e .
+```
+and 
+```
+<<_:a :b "4"^^xsd:integer >> :d :e .
+```
+and
+```
+<<_:a :c "4"^^xsd:int >> :d :e .
+```
+
+## Opaque Semantics
+
+In the opaque semantics
+```
+:b owl:sameAs :c .
+<<_:a :b "4"^^xsd:int >> :d :e .
```

Comment:
Lines like 50-51 here exemplify why I want to see the optional whitespace at *both* ends of any given enclosure. `<<_:a` *looks* like it is one term, rather than being expandable to `<< _:a`.

Further, in documentation like this whole page, I find comprehensibility to be *vastly* increased by adding optional whitespace such that the columns visually align from start to finish, which would make these two lines look like the following:
```
:b                            owl:sameAs :c .
<< _:a :b "4"^^xsd:int     >> :d         :e .
```
— which match up with line 59, which has the widest quoted triple as well as the widest object within a quoted triple —
```
<< _:a :b "4"^^xsd:integer >> :d         :e .
```

Reply:
Yes, good idea.  I've fixed (to be uploaded soon).